CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+Once you're all set up, you'll have to bundle the gems using bundler and start jekyll:
+
+```
+gem install bundler
+bundle install
+jekyll serve --watch
+
+```
+
+To see the site locally, just head to `http://localhost:4000` in your browser. If you are new to Jekyll, you can find more about it [here](http://jekyllrb.com/docs/home/). Otherwise, you're ready to go!
+
+## Reporting a bug
+
+Before reporting a bug, please check if it has [already been reported](https://github.com/rails-girls-summer-of-code/summer-of-code/issues?&q=is%3Aissue%20label%3Abug%20). If it hasn't and you want to file [a new bug report](https://github.com/rails-girls-summer-of-code/summer-of-code/issues/new), provide as many details as possible: How can the error be reproduced? What was the expected and what the actual behaviour? Can you provide an error message or a screenshot that illustrates the problem?  
+
+## Suggesting a new feature
+
+We are open to suggestions and ideas to improve the website or to contribute content for our [blog](http://railsgirlssummerofcode.org/blog/).  
+As with filing a bug report, before suggesting a new feature or giving feedback please check it has not been [suggested before](https://github.com/rails-girls-summer-of-code/summer-of-code/issues). And if it hasn't, feel free to share your ideas [on our issue tracker](https://github.com/rails-girls-summer-of-code/summer-of-code/issues/new)!
+
+## Contributing to an existing issue
+
+You can view all of the currently open issues [on our issue tracker](https://github.com/rails-girls-summer-of-code/summer-of-code/issues). Most are [labeled](#issue-labels), to give you an idea of the sort of contribution that is expected. 
+
+### Finding an issue
+
+A good starting point if you're new to open source is the list of `beginner-friendly` [issues](https://github.com/rails-girls-summer-of-code/summer-of-code/issues?q=is%3Aissue+is%3Aopen+label%3Abeginner-friendly). These are issues that don't require much prior knowledge on our program or our website to get started. 
+Before getting started, please check that no one else is working on the issue: this will be clear by the assigned user (if there is one) and by the issue's comments/discussion.   
+
+If it seems like no one is working on the issue, great! You've officially found an issue to work on. You can leave a comment mentioning you'd like to work on the issue. Feel free to also leave a comment with your questions if anything is unclear or if you need help: Is the issue still valid? Are there any more detailed specifications to the issue?  
+
+### Creating a pull request
+
+You can find very thorough information on Pull Requests in [GitHub's documentation](https://help.github.com/articles/creating-a-pull-request/).
+The most important thing to keep in mind is that all pull requests should be made against `gh-pages`, which is our main branch.
+
```

Comment:
Re: your question if the blog info should be added: I think it would be great to add a link here. MAybe like this: RailsGirls who want to submit a blog post, find detailed info in this link.
I know that it would have saved me some searching :-)